extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -32,11 +31,17 @@ import {cryptoFor} from '../../../src/crypto';
 import {xhrFor} from '../../../src/xhr';
 import {toggle} from '../../../src/style';
 
+// Register doc-service factory.
+AMP.registerServiceForDoc(
```

Comment:
The best option is to register the service like so:
```
AMP.registerServiceForDoc('amp-analytics-instrumentation', InstrumentationService)
```

And change `instrumentationServiceForDoc` to return the service and not create a new one.